src/Hooks/SeparateAddressFieldsHooks.php

```diff
@@ -347,6 +395,46 @@ public function extendSelectorsWithSeparateAddressFields(array $localeFields): a
             $this->createSelectorFor('fieldNumberSuffix')
         );
     }
+    /**
+     * Get POST data and validate it exists.
+     * @return array|null
+     */
+    private function getPostData(): ?array
+    {
+        $post = wp_unslash(filter_input_array(INPUT_POST));
+        return $post ?: null;
+    }
+
+    /**
+     * Get country code for the given address type from order or POST data.
+     * @param string $type 'billing' or 'shipping'
+     * @param WC_Order|null $order
+     * @param array|null $post
+     * @return string
+     */
+    private function getCountryCode(string $type, ?WC_Order $order = null, ?array $post = null): string
```

Comment:
In regel 226 return je early als `! $order`, dan kom je toch nooit in de $post logica? <- sanity check.